operator/internal/manifests/openshift/configure.go

```diff
@@ -212,5 +213,48 @@ func ConfigureOptions(configOpt *config.Options) error {
 		}
 	}
 
+	// Check user workload is enbaled.
+	if !uwam {
+		return nil
+	}
+
+	// Configure user-workload alertmanager when.
+	if len(configOpt.Overrides) == 0 {
+		configOpt.Overrides = map[string]config.LokiOverrides{}
+	}
+
+	lokiOverrides, ok := configOpt.Overrides[tenantApplication]
+	if !ok {
```

Comment:
You can unwrap the lines below by using the inverse condition check and do an early return, e.g.:
```go
if ok {
    return nil
}
```